examples/css3d_molecules.html

```diff
@@ -129,6 +129,7 @@
 
 				renderer = new CSS3DRenderer();
 				renderer.setSize( window.innerWidth, window.innerHeight );
+				renderer.domElement.style.outline = 'none';
```

Comment:
It seems this example does not need this code. Better to add it only where required.